CHANGELOG.rst

```diff
@@ -1,5 +1,11 @@
 Changelog
 =========
+v9.7.0
+------
+
```

Comment:
nit: let's remove this for now and add it when we change the version and make the release tag. Furthermore, I think it may be a smaller change maybe 9.6.1 since no functionality is being changed.